### DISCUSSION TITLE
How to handle multi-domain with the same node - nuxt instance?

### DISCUSSION THREAD

**Discussion by agracia-foticos** [nuxt/nuxt#16532] [Questions]
We are trying to manage with the same Nuxt project and with a single node instance, the site in "multi-domain" mode.

Each site will have its own private and public settings.

Can you think of any smart way to manage it?

**Comment by TheAlexLichter** [comment]
See https://github.com/nuxt/framework/discussions/6057 e.g. for subdomain handling.
